README.md

```diff
@@ -25,9 +25,26 @@ Usage: gitrise [options]
 
 Please see [here](https://api-docs.bitrise.io/#/) for Bitrise API Reference 
 
-## Tests
+## Contributing
+
+Bug reports and suggestions for improvement are always welcome! Pull requests are also accepted!
+
+If you are interested in adding functionality through a pull request, please open a new issue first so that we have the chance to discuss it first.
```

Comment:
"please open a new issue so that we have the chance to discuss it first", instead I having the word `first` mentioned twice.

Reply:
It's so funny, I was going to change that to exactly what you suggested...but must have gotten forgotten to do it. I changed it :P